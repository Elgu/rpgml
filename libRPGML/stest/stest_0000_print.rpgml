print( "Hello World!\n" );
print( 0.125 );
print( "\n" );
print( 23 );
print( "\n" );
