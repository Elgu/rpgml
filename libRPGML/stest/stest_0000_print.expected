Hello World!
0.125
23
